mobile/src/lib/cozy-helper.js

```diff
@@ -34,3 +34,16 @@ export const initBar = () => {
     lang: 'en'
   })
 }
+
+export const isClientRegistered = async (client) => {
+  return await cozy.client.auth.getClient(client).then(client => true).catch(() => false)
```

Comment:
I think that the argument in the catch callback is a `FetchError` and has a `status` field. Can you try this?

```js
.catch((err) => !(err.status && err.status >= 400) )
```